src/contributor-guide/dod/tco-analysis.md

```diff
@@ -0,0 +1,110 @@
+---
```

Comment:
Maybe the following question is already clarified somewhere but I could not find the answers. Who is a consumer of TCO analysis results? I mean, will this information be used to make any decisions? If yes, who, how and when will be making those decisions?

Reply:
The results of the TCO analysis should be used to ensure the value of the changes outweighs the costs for merchants. They are going to be used exactly the same way they are currently used, adding the analysis required to DoD does not include changes to the process.